test/automation/src/sql/notebook.ts

```diff
@@ -372,31 +372,32 @@ export class NotebookView {
 	 * @returns tree item ids from Notebooks View
 	 */
 	async getNotebookTreeItemIds(): Promise<string[]> {
-		return (await this.code.waitForElements(NotebookView.notebookTreeItem, false)).map(item => item.attributes['id']);
+		return (await this.code.waitForElements(NotebookTreeView.notebookTreeItem, false)).map(item => item.attributes['id']);
 	}
 
 	/**
 	 * Pin the first notebook in the Notebooks View
 	 */
 	async pinNotebook(): Promise<void> {
 		const notebookIds = await this.getNotebookTreeItemIds();
-		await this.code.waitAndDoubleClick(`${NotebookView.notebookTreeItem}[id="${notebookIds[0]}"]`);
-		await this.code.waitAndClick(`${NotebookView.notebookTreeItem}${NotebookView.selectedItem} .codicon-pinned`);
+		// Pinning SQL notebook to prevent the Configure Python Wizard from showing, since Python is no longer set up when the NotebookTreeView test suite starts
+		await this.code.waitAndDoubleClick(`${NotebookTreeView.notebookTreeItem}[id="${notebookIds[1]}"]`);
```

Comment:
I would suggest that instead you instead have the test select the item to pin and then call this function to avoid confusion when calling "pinNotebook" actually ends up pinning 2 separate Notebooks. (it's good to do whatever you can to avoid unexpected behavior and keep things as simple as possible).